configs/components/fesom/fesom-2.0.yaml

```diff
@@ -203,14 +216,40 @@ outdata_targets:
 
 choose_use_icebergs:
         True:
-            ib_num_old: 0
-            ib_num_new: 0
-
-            add_restart_in_files:
-                    icb_restart: icb_restart
-
-            add_restart_out_files:
-                    icb_restart: icb_restart
+            outputs: 
+                    [Av, Bo, Kv, MLD1, MLD2, N2, Redi_K, a_ice, alpha, atmice_x, atmice_y, atmoce_x, atmoce_y, beta, bolus_u, bolus_v, bolus_w, evap, fer_C, fer_K, fh, fw, hbl, iceoce_x, iceoce_y, m_ice, m_snow, prec, reso, runoff, salt, slope_x, slope_y, slope_z, snow, ssh, sss, sst, temp, tx_sur, ty_sur, u, uice, v, vice, vve, w, h2o16, h2o18, hDo16, h2o16_ice, h2o18_ice, hDo16_ice, ibfwe, ibfwl, ibfwb, ibfwbv, ibhf]
```

Comment:
The `outputs` variable does not seem to be useful anymore (it is not used anywhere in the `fesom-2.0.yaml`). See this:
```
212 outdata_sources:
213         fesom: "*.fesom.*.nc"
214 outdata_targets:
215         fesom: "*.fesom.*.nc"
```
@ackerlar , are you okay with me removing the `outputs` variable, or am I missing something?